packages/tools/examples/splineTools/index.ts

```diff
@@ -0,0 +1,278 @@
+import { RenderingEngine, Types, Enums } from '@cornerstonejs/core';
+import {
+  initDemo,
+  createImageIdsAndCacheMetaData,
+  setTitleAndDescription,
+  addDropdownToToolbar,
+  addSliderToToolbar,
+} from '../../../../utils/demo/helpers';
+import * as cornerstoneTools from '@cornerstonejs/tools';
+
+// This is for debugging purposes
+console.warn(
+  'Click on index.ts to open source code for this example --------->'
+);
+
+const DEFAULT_SPLINE_RESOLUTION = 20;
+const DEFAULT_CARDINAL_SCALE = 0.5;
+
+const {
+  SplineROITool,
+  PanTool,
+  ZoomTool,
+  ToolGroupManager,
+  Enums: csToolsEnums,
+} = cornerstoneTools;
+
+const { ViewportType } = Enums;
+const { MouseBindings } = csToolsEnums;
+const renderingEngineId = 'myRenderingEngine';
+const viewportId = 'CT_STACK';
+
+// ======== Set up page ======== //
+setTitleAndDescription(
+  'Spline Tools Stack',
+  'Spline tools for a stack viewport'
```

Comment:
please add more information here that you can change the spline type or use sliders etc. etc.